pyatv/interface.py

```diff
@@ -912,6 +912,7 @@ class DeviceInfo:
     MODEL = "model"
     RAW_MODEL = "raw_model"
     MAC = "mac"
+    AIRPLAY_IDENTIFIER = "airplay_id"
```

Comment:
I would prefer to rename this to `output_device` (and correspondingly change "AIRPLAY") to de-couple from AirPlay as a technology as much as possible. Theoretically we could get another technology in the future.